backend/pom.xml

```diff
@@ -90,6 +96,12 @@
             <type>pom</type>
             <scope>test</scope>
         </dependency>
+        <dependency> <!-- enables mocking of classes with Spock (in addition to interfaces) -->
+            <groupId>net.bytebuddy</groupId>
```

Comment:
Actually I dont know what this library does, could you explain it to me ahhaa sorry

Reply:
This library is used to test stubbing classes without using Spock. I have found a way to do the exact same thing with Spock now. I'll remove this dependency because I forgot to.